#!/usr/bin/env python

PACKAGE="green_robot"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("kPos", double_t, 0, "Position control gain", 1.0, 0.0, 2.0)
gen.add("kAngle", double_t, 0, "Angular control gain", 1.5, 0.0, 2.0)
#gen.add("xRef", double_t, 0, "X of reference position", 0.0, -2.0, 2.0)
#gen.add("yRef", double_t, 0, "Y of reference position", 0.0, -2.0, 2.0)

exit(gen.generate(PACKAGE, "green_robot","ControlPositionCFG"))
